### DISCUSSION TITLE
Add Resolution field to Query options

### DISCUSSION THREAD

**Discussion by adeverteuil** [grafana/grafana#52752] [Dashboards, Panels & Visualizations]
Some data sources like Prometheus and Loki have had additional query options like "Resolution" or "Min interval". The Min interval parameter was removed from the Loki query editor, and Resolution was removed from the Prometheus query editor. Furthermore, these data source specific options duplicate what Grafana does already in the Query options, and in some cases the implementation is buggy.

See this issue where this discussion started: https://github.com/grafana/grafana/issues/52290
A follow up discussion on the confusion around interval with the Loki data source: https://github.com/grafana/grafana/issues/52387

This request is to add a Resolution field into the panel Query options. This would restore a convenient functionality to control the granularity of a query and extend it to any data source. Any Min interval or Min step field could be removed from data source plugins to reduce duplication.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
